### ISSUE TITLE
Currently unavailable input devices make Pa_Initialize() fail. 

### ISSUE THREAD

**Issue by PortAudio-admin** [PortAudio/portaudio#235]
*Issue created by Anonymous on Assembla*

On Windows10 there are some input devices (SoundMAX Integrated Digital HD Audio) which for some reason are "Currently unavailable" (CD Player, Line In and Stereo Mix).

I don't need them but when Wasapi is initialized it *fails to call IAudioClient_GetDevicePeriod* on these devices, which returns 0x88890008, which should be AUDCLNT_E_UNSUPPORTED_FORMAT.
So basically one broken device stops the complete portaudio library from working.

**Comment by dmitrykos** [comment]
Fixed #235 already by this commit a525c38289b8c98554952caae792c2d1794d589c.
